src/core/directives/metadata_directives.ts

```diff
@@ -1225,3 +1225,38 @@ export class HostBindingMetadata {
 export class HostListenerMetadata {
   constructor(public eventName: string, public args?: string[]) {}
 }
+
+/**
+ * Interface for creating NgModuleMetadata
+ */
+export interface NgModuleMetadataType {
```

Comment:
yup I've looked at ng2 source code today at work. ugh.... 💨 I mean we can refactor things before 3.0 stable is out if there will be some additional time. 

And I'm no boss, it's open source! it's about opinions and open discussions :)